app/controllers/organizations_controller.rb

```diff
@@ -82,7 +82,8 @@ def reject
   private
 
   def set_organization
-    @organization = Organization.find_by_slug(params[:id])
+    @organization = Organization.find_by(slug: params[:id])
```

Comment:
Weird change.. I believe find_by_slug is generated automagically so it doesn't need to be implemented in model